FinalProject.iml

```diff
@@ -18,5 +18,7 @@
     <orderEntry type="library" name="sqlite-jdbc-3.32.3.2" level="project" />
     <orderEntry type="library" name="jackson-annotations-2.12.2" level="project" />
     <orderEntry type="library" name="RojeruSan" level="project" />
+    <orderEntry type="library" name="guava-30.1.1-jre" level="project" />
+    <orderEntry type="library" name="flatlaf-1.1.1" level="project" />
```

Comment:
FlatLaf apears twice in the project manifest, please checkout this @jwar28.

Reply:
ok i will solve it